### PR TITLE
Fixes #50 Added movie modal box functionallity

### LINKED ISSUE #50: Create modalbox for moviecards
## The user wants a modalbox where there contains more info about a movie
When a user clicks on a movie card on the homepage. The user wants to be redirected to a modal where there is som more information about the movie.
The modal should fill the whole width/height of the page.

### PR DESCRIPTION
**Pull request by marcusmoller97**
* Fixed **index.html**
* Added click event to movie card (opens movie modal when clicked)
* Added style to movie modal box

### CODE REVIEW

**Review by NordicNomadicLife — APPROVED** [review]
looks good

**Review by EmpyreanMist — APPROVED** [review]
Well done! Pressing anywhere on the cards will open the modal and cover full screen.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules
-main.css.map
\ No newline at end of file
+main.css.map
+.vscode
```
Comment (itsJoel59) [inline .gitignore:3]: Vad gör .vscode? Aldrig sett förut 🧐

**Review by itsJoel59 — APPROVED** [review]
Nice work, I like the functions you made. Good work!

File: .gitignore (hunk 1 above)
Comment (marcusmoller97) [reply .gitignore:3]: Får upp en vscode fil med live server ibland. Är ingenting som behöver följa med.

File: index.html [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+  <head>
+     <meta charset="utf-8" />
+     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+     <title>Kino</title>
+     <meta name="viewport" content="width=device-width, initial-scale=1" />
+     <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+     <script type="module" src="main.js" defer></script>
+     <script
+       src="https://kit.fontawesome.com/8357c99e29.js"
+       crossorigin="anonymous"
```
Comment (itsJoel59) [inline index.html:12]: Cool, I like the use of _crossorigin_ attribute as well as _defer_

**Review by itsJoel59 — COMMENTED** [review]
Can't see anything wrong with index.html, good job!

**Pull request merged**